packages/@apphosting/adapter-nextjs/src/utils.ts

```diff
@@ -1,21 +1,27 @@
+import { spawnSync } from "child_process";
 import fsExtra from "fs-extra";
-import { PHASE_PRODUCTION_BUILD } from "./constants.js";
-import { ROUTES_MANIFEST } from "./constants.js";
+import { createRequire } from "node:module";
+import { join, relative, normalize } from "path";
 import { fileURLToPath } from "url";
-import { OutputBundleOptions } from "./interfaces.js";
 import { stringify as yamlStringify } from "yaml";
-import { spawnSync } from "child_process";
 
-import { join, relative, normalize } from "path";
+import { PHASE_PRODUCTION_BUILD } from "./constants.js";
+import { ROUTES_MANIFEST } from "./constants.js";
+import { OutputBundleOptions, RoutesManifest } from "./interfaces.js";
+import { NextConfigComplete } from "next/dist/server/config-shared.js";
 
-import type { RoutesManifest } from "./interfaces.js";
 // fs-extra is CJS, readJson can't be imported using shorthand
 export const { move, exists, writeFile, readJson } = fsExtra;
 
-export async function loadConfig(cwd: string) {
+export const DEFAULT_COMMAND = "npm";
```

Comment:
let's make sure we have java docs for all public/exported functions and especially constants